### ISSUE TITLE
Is there any way to increase font size?

### ISSUE THREAD

**Issue by krab** [Openarl/PathOfBuilding#48]
I mean fonts in all application, everything so small, and current font with white color looks ugly on typical 96dpi displays.

**Comment by Openarl** [comment]
Not easily. The entire UI would need tweaking to accommodate a larger text size.

This is first complaint I've had regarding the text size. What's your resolution and screen size? The text looks fine to me at 1920x1080 on a 27" monitor.

Regardless, one factor that's probably contributing is that the current text renderer used by the program doesn't support ClearType (or unicode, for that matter). Some day I hope to switch it out for a FreeType-based renderer, which should look much better, even at the same text size. That'll be quite an undertaking though.

**Comment by krab** [comment]
2048x1152 at 23"

>doesn't support ClearType 

Clear enough. ClearType should fix all font rendering problems on lcd display.

> Not easily. The entire UI would need tweaking to accommodate a larger text size.

This probably means program don't support HiDPI either?

**Comment by Openarl** [comment]
> This probably means program don't support HiDPI either?
Correct. The program isn't really designed in a way that allows me to support something like that.

Since you're having this issue, I'll try to find the time to look more closely at FreeType. I need to switch to that at some point anyway since I want to add unicode support.

**Comment by rbenson** [comment]
I want to join in and say the font size is a bit too small for me. I just found PoB over the weekend

**Comment by jzakariya** [comment]
Add my vote too please. Using a 2560 x 1440 display.

**Comment by SimpleCookie** [comment]
Add my vote as well. It's super tiny on 2560x1440p, it's a pain for newbie players like me :)

**Comment by bjbeare** [comment]
+1 on a 4k (somewhat small) secondary display it is too small to read. To be sure, this is a 'first world problem" but it would be great to be able to tweak the UI text sizes

**Comment by omgirok** [comment]
+1 to increasing font size please!

**Comment by defekT1312** [comment]
Pretty Please

**Comment by BenyaminJeizan** [comment]
+1

**Comment by direded** [comment]
+ for 2k display

**Comment by soulcube** [comment]
up for 2k 27" Samsung Odyssey, also this is being run with nvidia graphics.
